README.md

```diff
@@ -692,3 +693,4 @@ great community of open source
 * Vasiliy Ermolovich
 * Andrew Haines
 * Sean Linsley
+* Cliff Braton (cliff.braton@gmail.com)
```

Comment:
@codebycliff
Is it OK to move you to the "Historical maintainers"? If you are still interested in maintaining `draper`, please tell us 🙏

Reply:
My decision is based on no replies from @codebycliff for a while, sadly.

@codebycliff, we still need you to update the gem (when we are ready to).